api/invite.go

```diff
@@ -485,14 +467,42 @@ func (a *Api) AcceptTeamInvite(res http.ResponseWriter, req *http.Request, vars
 		return
 	}
 
+	switch conf.Type {
+	case models.TypeMedicalTeamPatientInvite:
+		a.acceptTeamInvite(res, req, conf)
+	case models.TypeMedicalTeamInvite:
+		a.acceptTeamInvite(res, req, conf)
```

Comment:
I think we can use a case list in go 
```suggestion
	case models.TypeMedicalTeamPatientInvite, models.TypeMedicalTeamInvite :
		a.acceptTeamInvite(res, req, conf)
```

Reply:
cool :+1: